config/agent.yaml

```diff
@@ -0,0 +1,16 @@
+control_plane:
```

Comment:
we still need this file?

Reply:
Ah, we don't use this now, let me remove this. But using a config file as metadata for the service also a good idea, we'll think about this later.